### PR TITLE
Feat/sc 15 category page text

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-15

### CODE REVIEW

File: scripts/convert.js [hunk 1]
```diff
@@ -1,25 +1,59 @@
...
+        } else {
+          // if the indicator source (column name) doesn't exist in the csv, throw an error
+          throw new Error(`Key not found! ${indicator.source}`);
+        }
+      });
+
+      return city;
+    });
+
+    process.stdout.write(JSON.stringify(cities, null, 2)); // pretty-print
```
Comment (davidgilbertson) [inline scripts/convert.js:57]: Looks like a big change but really it's just checking for types now.

File: src/style/_dimensions.scss [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 $z-index-main-app: 1;
 $z-index-left-nav: 2;
+
+@mixin container {
+  max-width: $app-max-width;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 16px;
+  padding-right: 16px;
+}
```
Comment (davidgilbertson) [inline src/style/_dimensions.scss:17]: You can use this anywhere you would have used the bootstrap `container` class.

File: scripts/convert.js [hunk 3]
```diff
@@ -1,25 +1,59 @@
 #!/usr/bin/env node -r @std/esm
 
 // This file relies on ES6 modules
-// so must be run via "npm run convert" which uses @std/esm
+// so must be run via "npm run generate-city-data" which uses @std/esm
```
Comment (micapam) [inline scripts/convert.js:4]: Not strictly true: you can just run the file like any other executable

**Review by micapam — APPROVED** [review]
lgtm

**Pull request merged**